docs/data-layer-api.md

```diff
@@ -0,0 +1,281 @@
+# Data Layer API Reference
+
+The [Data Layer API](/api-docs/store-management/data-layer-api) is a GraphQL management API that allows you to enable and disable the [Big Open Data Layer](/api-docs/analytics/bodl-for-storefronts) for a store.
+
+**Request URL:** 
+
+`https://api.bigcommerce.com/stores/{store_hash}/v3/...`
+
+**Path Parameters:**
+
+| Name | Type | Description | Required |
+| :--- | :--- | :--- | :--- |
+| `store_hash` | string | Unique ID for a store | Yes |
+
+## Authentication
+
+### OAuth scopes
+
+| UI Name | Permission | Parameter |
+| :--- | :--- | :--- |
+|  | modify |  |
```

Comment:
@solofeed I assume the GraphQL DataLayer API authenticates using Store API Accounts (like the management APIs that are REST APIs)? Do you have the UI name and parameter for the OAuth Scopes?

Reply:
scope - Information
permission - Manage Web Analytics